test/integration/particle_emitter.cc

```diff
@@ -120,6 +120,10 @@ TEST_F(ParticleEmitterTest, SDFLoad)
                 // and let rendering do the findFile instead
                 EXPECT_EQ(std::string(),
                     _emitter->Data().color_range_image().data());
+
+                // particle scatter ratio is temporarily stored in header
+                EXPECT_DOUBLE_EQ(0.01,
+                  math::parseFloat(_emitter->Data().header().data(0).value(0)));
```

Comment:
Would it be possible to retrieve the data from the header via a key ("particle_scatter_ratio") instead of an index? That would make this check less failure-prone in the future if we had to modify how/what data is stored in the header.

Reply:
yep done. 0718e8980691152abd258a8370b1728ca73f17ca